### ISSUE TITLE
[Bug][A11Y][Phone number input] On Android, keyboard focus is not visible on country selector

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#856]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S25

### OS Version

Android 16

### App version

2.0.0 (710)

### Describe the issue

on Android, keyboard focus is not visible on country selector. 
<img width="107" height="64" alt="" src="https://github.com/user-attachments/assets/ae8546b6-afe1-4a3c-87fa-842ed5188d8b" />

### Expected Behavior

We should at least see the default darker grey focus, like the focused input:

<img width="384" height="96" alt="" src="https://github.com/user-attachments/assets/876a420c-3e81-4cef-8713-12adcb946524" />
